scripts/helpers.ts

```diff
@@ -113,8 +114,9 @@ export const setupOrRetrieveGateway = async (
         OVM_L1ERC20Gateway = newGateway.OVM_L1ERC20Gateway
         OVM_L2DepositedERC20 = newGateway.OVM_L2DepositedERC20
     } else {
-        OVM_L1ERC20Gateway = new Contract(l1ERC20GatewayAddress, Def__MyL2DepositedERC20.abi, l1Wallet)
-        const l2ERC20GatewayAddress = await OVM_L1ERC20Gateway.l2ERC20Gateway()
+        const OVM_L1ERC20Gateway_Def = getContractDefinition('OVM_L1ERC20Gateway', false);
```

Comment:
To avoid the confusion thought readers may get, I think it's best to get the name of the contract definition the same, as `Def__MyL2DepositedERC20`